crates/holochain_metrics/src/dashboards/wasm.json

```diff
@@ -0,0 +1,182 @@
+            }
+          ],
+          "staticLegend": {},
+          "tableOptions": {
+            "sortBy": "_value",
+            "verticalTimeAxis": true
+          },
+          "timeFormat": "HH:mm:ss.sss",
+          "width": 5
+        },
+        {
+          "colors": [
+            {
+              "id": "base",
+              "name": "white",
+              "type": "text",
+              "hex": "#ffffff"
+            }
+          ],
+          "fieldOptions": [
+            {
+              "displayName": "time",
+              "fieldName": "_time",
+              "visible": true
+            },
+            {
+              "displayName": "value",
+              "fieldName": "_value",
+              "visible": true
+            },
+            {
+              "displayName": "_start",
+              "fieldName": "_start"
+            },
+            {
+              "displayName": "_stop",
+              "fieldName": "_stop"
+            },
+            {
+              "displayName": "_field",
+              "fieldName": "_field"
+            },
+            {
+              "displayName": "_measurement",
+              "fieldName": "_measurement"
+            },
+            {
+              "displayName": "agent",
+              "fieldName": "agent"
+            },
+            {
+              "displayName": "dna",
+              "fieldName": "dna"
+            },
+            {
+              "displayName": "fn",
+              "fieldName": "fn",
+              "visible": true
+            },
+            {
+              "displayName": "zome",
+              "fieldName": "zome",
+              "visible": true
+            },
+            {
+              "displayName": "mean_value",
+              "fieldName": "mean_value",
+              "visible": true
+            },
+            {
+              "displayName": "stat_type",
+              "fieldName": "stat_type",
+              "visible": true
+            }
+          ],
+          "height": 7,
+          "kind": "Table",
+          "name": "All function calls",
+          "queries": [
+            {
+              "query": "from(bucket: \"windtunnel\")\n  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)\n  |> filter(fn: (r) => r._measurement == \"hc.ribosome.wasm.usage\")\n  |> filter(fn: (r) => r[\"dna\"] + \"___\" + r[\"agent\"] == v.CellId)\n  |> filter(fn: (r) => \n    if exists r[\"run_id\"] then r[\"run_id\"] == v.RunId\n    else true\n  )"
+            }
+          ],
+          "staticLegend": {},
+          "tableOptions": {
+            "sortBy": "_value",
+            "verticalTimeAxis": true
+          },
+          "timeFormat": "HH:mm:ss.sss",
+          "width": 7,
+          "xPos": 5
+        }
+      ],
+      "description": "Wasm usage per zome function",
+      "name": "Wasm Usage"
```

Comment:
New dashboard to display `hc.ribosome.wasm.usage`

Reply:
Yep, that metric was added for Holo after the dashboards were created.